### PR TITLE
Add proxy example format

### PR DESCRIPTION
**Pull request by cmedina003**
Added comments for the proxy values under serverMeta.

### CODE REVIEW

File: deploy/kubeturbo/values.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ serviceAccountName: "turbo-user"
 serverMeta:
   version: "8.0"
   turboServer: https://Turbo_server_URL
-#  proxy: http://username:password@proxyserver:proxyport
+#  proxy: http://username:password@proxyserver:proxyport or http://ip:port
```
Comment (ading1977) [inline deploy/kubeturbo/values.yaml:34]: `http://proxyserver:proxyport` ?

File: deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml [hunk 2]
```diff
@@ -14,6 +14,9 @@ spec:
     version: "8.1"
     turboServer: https://Turbo_server_URL
 
+  # URL format for Authenticated and Non-Authenticated
+  #   proxy: "http://username:password@ip:port or http://ip:port"
```
Comment (ading1977) [inline deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:18]: `proxyserver:proxyport` ?

Comment (cmedina003) [reply deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:18]: are you suggesting to use those terms?

Comment (ading1977) [reply deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:18]: Yes, to make them consistent across the code base.

**Review by ading1977 — APPROVED** [review]

**Pull request merged**